tests/operations.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitAndStartInstance,
+  visitAndStopInstance,
+} from "./helpers/instances";
+import {
+  createProfile,
+  deleteProfile,
+  randomProfileName,
+} from "./helpers/profile";
+import { assertTextVisible } from "./helpers/permissions";
+import {
+  checkNotificationExists,
+  dismissNotification,
+} from "./helpers/notification";
+
+let instance = randomInstanceName();
+let profile = randomProfileName();
+
+test.beforeAll(async ({ browser, browserName }) => {
+  instance = `${browserName}-${instance}`;
+  profile = `${browserName}-${profile}`;
+  const page = await browser.newPage();
+  await createProfile(page, profile);
+  await createInstance(page, instance);
+  await page.close();
+});
+
+test.afterAll(async ({ browser }) => {
+  const page = await browser.newPage();
+  await deleteProfile(page, profile);
+  await page.close();
+});
```

Comment:
We probably don't need the before and after for this test. Also, the profile is not needed. The only relevant line is with create instance. We can add that as a first thing to do in the test below.